.github/workflows/release.yml

```diff
@@ -10,10 +10,15 @@ jobs:
     name: Check release required
     if: github.event.pull_request.merged && endsWith(github.repository, '-private') != true
     runs-on:
-      group: Default
+      group: macos-gh
     outputs:
       release: ${{ steps.check.outputs.ready }}
     steps:
+      - name: Install JDK
```

Comment:
I am wondering why we didn't have this step before and now is needed?

Reply:
I don't know, maybe the default GH runner has Java preinstalled, but our mac os one doesn't